src/FSharpSource.Settings.targets

```diff
@@ -25,11 +25,10 @@
 
   <!-- Standard interpretations of Debug and Release configurations -->
   <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
-    <DebugType Condition=" '$(DebugType)' == '' ">full</DebugType>
+    <DebugType Condition=" '$(DebugType)' == '' ">portable</DebugType>
```

Comment:
VS2014 and Dev15 from Update 2 onwaqrds can both debug portable PDB's, and portable PDBs are much ... much smaller.